src/components/legend.ts

```diff
@@ -144,13 +164,17 @@ export module Components {
 
       let availableWidthForEntries = Math.max(0, (availableWidth - this._padding));
 
-      let entryNames = this._colorScale.domain().slice();
-      entryNames.sort(this.comparator());
+      let entryLabels: d3.Map<string> = d3.map<string>();
+      this._colorScale.domain().slice().forEach((d) => {
+        entryLabels.set(this._formatter(d), d);
```

Comment:
To get computed value from raw value, I can use function formatter(value). In this case, I want to get raw value from computed value. That's why I want to use a map to store those values.